packages/web/scripts/security.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+tempFile=$(mktemp)
+cat > "${tempFile}" <<EOF
+
+        add_header X-Frame-Options "SAMEORIGIN";
+        add_header X-Content-Type-Options "nosniff";
+        add_header Referrer-Policy "strict-origin-when-cross-origin";
+        add_header Content-Security-Policy "default-src 'self'; script-src 'self'; style-src 'self' 'unsafe-inline'; frame-ancestors 'self'; connect-src 'self' http://${API_DOMAIN} https://${API_DOMAIN} ws://${API_DOMAIN} wss://${API_DOMAIN} blob:;";
+EOF
+
+if [ "${ENABLE_SECURITY_HEADERS}" = "true" ]; then
+  if grep -q "#SecurityHeaders" /etc/nginx/nginx.conf.tmpl; then
+    sed -i "/#SecurityHeaders/r ${tempFile}" /etc/nginx/nginx.conf.tmpl
```

Comment:
Is it correct?
* There is missing closing `/` for sed expression
* There is file name prepended by `r ` provided instead of actuall contents

Reply:
The `sed` command does not require a closing / for the search pattern when using the r (read from file) command. The use of `r` followed by a space and a filename inserts the content of the file after the line where the pattern matches.

Tested multiple times and it works right..